### PR TITLE
[Connectors][Addnote][1633257]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Added note.

### CODE REVIEW

File: msteams-platform/webhooks-and-connectors/how-to/connectors-using.md [hunk 1]
```diff
@@ -127,10 +127,12 @@ The `ActionCard` action supports three input types:
 
 If you want a multiselect list initially displayed in the compact style, you must specify both `"isMultiSelect": true` and `"style": true`.
 
+For all other details about Connector card actions, see **[Actions](/outlook/actionable-messages/card-reference#actions)** in the actionable message card reference.
```
Comment (KirtiPereira) [inline msteams-platform/webhooks-and-connectors/how-to/connectors-using.md:130]:
```suggestion
For more information on Connector card actions, see **[Actions]**(/outlook/actionable-messages/card-reference#actions) in the actionable message card reference.
```

File: msteams-platform/webhooks-and-connectors/how-to/connectors-using.md [hunk 2]
```diff
@@ -127,10 +127,12 @@ The `ActionCard` action supports three input types:
...
+
 > [!NOTE]
 > Specifying `compact` for the `style` property in Microsoft Teams is the same as specifying `normal` for the `style` property in Microsoft Outlook.
 
-For all other details about Connector card actions, see **[Actions](/outlook/actionable-messages/card-reference#actions)** in the actionable message card reference.
+> For HttpPOST action, the bearer token included with all requests includes the Azure AD identity of the Office 365 user who took the action.
```
Comment (KirtiPereira) [inline msteams-platform/webhooks-and-connectors/how-to/connectors-using.md:135]:
```suggestion
> For the HttpPOST action, the bearer token is included with the requests. This token includes the Azure AD identity of the Office 365 user who took the action.
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@Rajeshwari-v - please incorporate the changes.

**Review by Wajeed-msft — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR

**Review by KirtiPereira — APPROVED** [review]
Approved

**Pull request merged**